src/pages/HistoryPage/getHistoryData.ts

```diff
@@ -32,9 +36,24 @@ export const getHistoryData = async (
                 {
                   where: { start_date: { _gt: 'now' } },
                   order_by: [{ start_date: order_by.asc }],
-                  limit: 1,
                 },
-                { start_date: true, end_date: true },
+                {
```

Comment:
@levity  IApiEpoch is only used by history page and epoch form now. can I update it or it might be needed in the future?

Reply:
It's still used in `api.manifest.d.ts`

Reply:
yes but the extra elements are not needed there too. ok I will leave it unchanged and do it in your suggested ways

Reply:
it's fine to remove them as long as you do the extra testing to make sure there's no impact

Reply:
generally though the best thing is to use types derived from the Zeus graphql queries, instead of trying to use the same type across different contexts